content/articles/how-long-to-issue-ssl-certificate.markdown

```diff
@@ -9,10 +9,28 @@ categories:
 
 The time required to issue a new [SSL certificate](/articles/ssl-certificates) depends on many factors. The SSL certificate type, the validation process, and how quickly you respond with requested information from us or the [certificate authority](/articles/what-is-certificate-authority) communications.
 
-In general, issuing a new domain-validated SSL certificate can take from a few minutes to a few hours, **after you [approve the SSL certificate](/articles/ssl-certificates-email-validation)**.
+## Standard certificates
+
+For standard [single-name](/articles/ssl-certificates#standard-singlename) and [wildcard](/articles/ssl-certificates#standard-wildcard) certificates, it can take from a minimum of one hour to several hours, **after you [approve the SSL certificate](/articles/ssl-certificates-email-validation)**.
+
+Occasionally, the issuance may take longer and require up to several days. This is the case when some issue occurs during the issuance or validation. Common issues are:
+
+- misconfiguration of the domain
+- CAA records preventing the validation and issuance
+- issues in the email approval
 
 <warning>
 #### Remember to approve the certificate!
 
 In most cases, the purchase process can get stuck because the certificate is never approved by the owner. Once you submit the certificate, please monitor the approval email inbox and make sure to **click on the link contained in the email sent from the Certificate Authority** in order to validate and approve the certificate. Be sure to monitor your spam folder in case these emails are accidentally marked as spam.
 </warning>
+
+## Let's Encrypt certificates
+
+For [Let's Encrypt](/articles/ssl-certificates#letsencrypt) certificates the issuance is generally between 30 minutes and 1 hour.
+
+If the order is taking longer, it is generally a synthom or some issuance issue such as:
```

Comment:
Not sure what ‘a synthom or some issuance issue’ means...

Consider rephrase:
‘... an issuance problem such as:’ to avoid similar words and confusion.

Reply:
Actually, it was supposed to be _of_: _it is generally a synthom or some issuance issue such as_. I will reword it.